WebPhapp/backend/server.js

```diff
@@ -219,6 +272,24 @@ app.post('/api/v1/prescriptions/add',(req,res) => {
     return finish("TODO: build prescription add to blockchain", true);
 });
 
+// An api endpoint that cancels the prescription associated with a
+// given prescription ID.
+// example: http://localhost:5000/api/v1/prescriptions/cancel/2
+app.get('/api/v1/prescriptions/cancel/:prescriptionID', (req,res) => {
```

Comment:
Why is this a GET and not a POST?

Reply:
Actually I was looking at the wrong version. Question has changed. What happened to the stubbed route for `/api/v1/prescriptions/cancel/` in the final version of your PR?

Reply:
Huh? Yeah, upon merging I must have accidentally deleted that. Good catch!